Specs/Number/Arabic/NumberRangeModel.json

```diff
@@ -1291,6 +1281,366 @@
+    "Results": [
+      {
+        "Text": "أعلى من العاشر ولكنه أقل من الخامس عشر",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(10,15)"
+        },
+        "Start": 15,
+        "End": 52
+      }
+    ]
+  },
+  {
+    "Input": "هذا الرقم هو أعلى أو يساوي مائة أو أقل أو يساوي ثلاثمائة",
+    "NotSupportedByDesign": "java, javascript, python",
+    "NotSupported": "dotnet",
+    "Results": [
+      {
+        "Text": "أعلى أو يساوي مائة أو أقل أو يساوي ثلاثمائة",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[100,300]"
+        },
+        "Start": 13,
+        "End": 55
+      }
+    ]
+  },
+  {
+    "Input": "نطاق الرقم من ٢٠ إلى ١٠٠",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "من ٢٠ إلى ١٠٠",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[20,100)"
+        },
+        "Start": 11,
+        "End": 23
+      }
+    ]
+  },
+  {
+    "Input": "وهو أكثر من ثلاثين سنة من عمره",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "أكثر من ثلاثين",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(30,)"
+        },
+        "Start": 4,
+        "End": 17
+      }
+    ]
+  },
+  {
+    "Input": "يوجد حوالي خمسمائة أو أكثر في هذه المنتجات.",
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "خمسمائة أو أكثر",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[500,)"
+        },
+        "Start": 11,
+        "End": 25
+      }
+    ]
+  },
+  {
+    "Input": "ارتفاعه أقل من ١٧٠",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "أقل من ١٧٠",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,170)"
+        },
+        "Start": 8,
+        "End": 17
+      }
+    ]
+  },
+  {
+    "Input": "درجته ٢٠٠ أو أكبر",
+    "IgnoreResolution": true,
+    "NotSupportedByDesign": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "٢٠٠ أو أكبر",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[200,)"
+        },
+        "Start": 6,
+        "End": 16
+      }
+    ]
+  },
+  {
+    "Input": "درجته أقل من ٣٠ او ما يعادله",
+    "NotSupportedByDesign": "java, javascript, python",
+    "NotSupported": "dotnet",
+    "Results": [
+      {
+        "Text": "أقل من ٣٠ او ما يعادله",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,30]"
+        },
+        "Start": 6,
+        "End": 27
+      }
+    ]
+  },
+  {
+    "Input": "درجته تساوي ٣٠ أو ما تقل عنه",
+    "NotSupportedByDesign": "java, javascript, python",
+    "NotSupported": "dotnet",
```

Comment:
Why are these new cases not supported?

Reply:
@tellarin it was for our Neudesic branch. He (@MuhammedSalamneu) raised to wrong target by mistake.
@MuhammedSalamneu please close this PR.